lib/index.js

```diff
@@ -143,7 +144,7 @@ function stringify(obj) {
   var ret = {};
   for (var prop in obj) {
     if (obj.hasOwnProperty(prop)) {
-      ret[prop] = obj[prop].toString();
+      if (obj[prop]) ret[prop] = obj[prop].toString();
```

Comment:
This will drop any falsy values (`false`, `0`, `""`). We could just put an `obj = reject(obj)` above the for loop instead